deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -260,8 +425,13 @@ services:
       STORE_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
 
       STORE_GRPC_ADDR: 0.0.0.0:9460
+
+      OCIS_JWT_SECRET: ${OCIS_JWT_SECRET}
+      STORAGE_TRANSFER_SECRET: ${STORAGE_TRANSFER_SECRET}
+      OCIS_MACHINE_AUTH_API_KEY: ${OCIS_MACHINE_AUTH_API_KEY}
```

Comment:
according to https://owncloud.dev/extensions/store/configuration/, these config options do not exist for the store extension